docs/application-connector/10-02-application-gateway-troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Application Gateway troubleshooting
+type: Troubleshooting
+---
+
+In case user calls registered service and receives an error:
+
+- Verify that the call reached Application Gateway.  
+  To do that fetch logs from Application Gateway pod:
+  ```
+  kubectl -n kyma-integration logs -l app={APP_NAME}-application-gateway -c {APP_NAME}-application-gateway
+  ```
+  If the request reached the pod, it should be logged by Application Gateway.
```

Comment:
pod -> Pod

If it reached the Pod, why SHOULD it be logged? If it reaches then it's logged. 
If I got an error and the call's logged then what?